status/statusgo_backend_new/stickers.nim

```diff
@@ -0,0 +1,74 @@
+import json, stint, web3/[ethtypes, conversions], strutils, tables, sequtils
+import
+  ../types/[setting],
```

Comment:
@jrainville all lines here from 3 to 11 rely on the "old" `status-lib` code, and that's not the point here. `statusgo_backend_new` part is created with an idea to have here only new logic (actually calls to `status-go`) and nothing else. Cause at the end of refactoring phase we will remove everything else and only files under `statusgo_backend_new` will make `status-lib`.